account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -178,8 +179,7 @@ def populate(self):
     )
     def move_line_filters_change(self):
         domain = self._prepare_move_line_domain()
-        res = {"domain": {"move_line_ids": domain}}
-        return res
+        self.move_line_domain = domain
```

Comment:
Do it a computed non stored field instead of this.

Reply:
Updated